build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.0'
+        classpath 'com.android.tools.build:gradle:3.3.2'
```

Comment:
Can we use the version.gradle file to keep the version number and use from there?

Reply:
According your feedback now all of our hard coded gradle dependency library calling process has been switched to version.gradle